src/_components/Layouts/ControlBar.stories.tsx

```diff
@@ -216,7 +221,7 @@ export const APIError = Template.bind({});
 APIError.args = {
   collection: dummyCollection,
   context: { ...dummyCollection, branch: 'branch-1' },
-  branches,
+  branches: branches.map((branch) => branch.name),
```

Comment:
branches were mapped to string[] as the name seemed like the only thing required in the component. It may it simpler to add types to the component